src/style.css

```diff
@@ -0,0 +1,65 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: RobotoMedium;
+  font-weight: 500;
+  src: url(./fonts/Roboto-Medium.ttf);
+}
+
+body {
+  margin: 0;
+  --action-color: #00acdc;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 1150px;
+  margin: 0 auto;
+  margin-left: 50px;
+}
+
+.header {
+  display: flex;
+  margin-bottom: -1px;
+  font-family: "RobotoMedium", sans-serif;
+  border-bottom: 1px solid #f3f3f3;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-wrap {
+  display: flex;
+  margin-top: 4px;
+  margin-right: 50px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  margin-right: 30px;
```

Comment:
Instead of usig this margin + 20px mrgin of your last `.nav_link` element you should use horizontal paddings 50px for your header and reset the `margin-right` for your `.nav-link:last-child`